python/lsst/analysis/ap/utilities.py

```diff
@@ -0,0 +1,70 @@
+# This file is part of analysis_ap.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["makeSimbadLink"]
+
+from IPython.display import display, Markdown
+import astropy.coordinates as coord
+from astroquery.simbad import Simbad
+import astropy.units as u
+
+
+def makeSimbadLink(source, arcsec_radius=3.0, make_table=False):
+    """Search Simbad for associated sources within a 3 arcsecond region
+        Maximum search radius should not be greater than 5 arcseconds.
```

Comment:
Why the restriction on search radius?  I know it's not very physical for larger radii, but that's not a restriction of the Simbad query, correct?

I'd document this with the radius argument below, and if you think a cut is needed maybe consider adding a check and warning in the code?

Reply:
The restriction was based on the radius of the cutout, which is ~5 arcseconds. However, you are right and it isn't necessarily needed. A warning would probably be enough.